lib/maze/capabilities/devices.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module Maze
+  module Capabilities
+    # Provides a source of capabilities used to run tests against specific BrowserStack devices
+    # noinspection RubyStringKeysInHashInspection
+    class Devices
+      APPIUM_1_6_5 ||= '1.6.5'
+      APPIUM_1_7_2 ||= '1.7.2'
+      APPIUM_1_8_0 ||= '1.8.0'
+      APPIUM_1_15_0 ||= '1.15.0'
```

Comment:
Ah, didn't see this when I wrote my first response.  These were put in some time ago and the diff here is just misleading.  I'm not sure if there is another or better way of doing it and it's what I've seen on various SO posts - open to other suggestions, of course.

Reply:
I think the problem was Cucumber requiring these files — https://github.com/bugsnag/maze-runner/blob/fd8ce09c7ab913afab69bf3fe17a48a1613254e3/bin/maze-runner#L64-L66

We should be able to remove `lib/maze` from the list of paths and load those paths up-front if necessary (either in `bin/maze-runner` or `lib/maze.rb`)

Reply:
Ah... great!  This is starting to come together now.  I've removed those lines, added a load of `require`s at the top of `bin/maze-runner` and moved `hooks.rb` back under features (because it's Cucumber specific).

There's probably still room for improvement, but it's no longer necessary to use `||=`.